packages/xchain-bitcoincash/src/utils.ts

```diff
@@ -110,3 +112,143 @@ export const parseTransaction = (tx: Transaction): Tx => {
     hash: tx.txid,
   }
 }
+
+/**
+ * Validate the BCH address.
+ *
+ * @param {string} address
+ * @param {Network} network
+ * @returns {boolean} `true` or `false`.
+ */
+export const validateAddress = (address: string, network: Network): boolean => {
+  try {
+    bitcash.Address.fromString(address, bchNetwork(network))
+    return true
+  } catch (error) {
+    return false
+  }
+}
+/**
+ * Scan UTXOs from sochain.
+ *
+ * @param {string} clientUrl sochain Node URL.
+ * @param {Address} address
+ * @returns {Array<UTXO>} The UTXOs of the given address.
+ */
+export const scanUTXOs = async (clientUrl: string, address: Address): Promise<bitcash.Transaction.UnspentOutput[]> => {
+  const unspents = await getUnspentTransactions({ clientUrl, address })
```

Comment:
What happened if `getUnspentTransactions` failed?

Reply:
If it's failed, it will throw an error, Let me add try...catch for the API requests.